packages/next/package.json

```diff
@@ -21,6 +20,10 @@
   "files": [
     "dist"
   ],
+  "dependencies": {
+    "@vercel/nft": "0.22.5",
+    "esbuild": "0.12.22"
```

Comment:
`esbuild` should be devDependencies, right?

Reply:
I was marked as `--external` when invoking ncc in the original build script. I assume it's used when compiling edge functions?

Reply:
That’s odd. How would it get installed then if it’s marked as external but not in dependencies? Was it luck that yarn would hoist it?

Reply:
Actually I think it's not used. Will move back to dev deps.